packages/use-contractkit/src/use-contract-kit-methods.ts

```diff
@@ -79,13 +79,21 @@ export function useContractKitMethods(
         initialisedConnector.onAddressChange?.((address) => {
           dispatch('setAddress', address);
         });
-        return initialisedConnector;
       } catch (e) {
+        if (typeof e === 'symbol') {
```

Comment:
I really don't have a good idea as to what's going on here. Do you think you could make a comment explaining why we would catch a Symbol?

Reply:
Yeah. I wasn't too sure about that one but I wanted to make sure we can identify user cancellation easily. I declared a `
`export const CANCELED = Symbol('canceled');` in `packages/walletconnect-v1/src/constants.ts`

Reply:
It would be, however it means use-contract-kit-methods specifically depends on `wallect-connect-v1` ? kinda weird